charts/csi-powerstore/templates/controller.yaml

```diff
@@ -39,28 +42,31 @@ rules:
     verbs: ["get", "list", "watch"]
   {{- end }}
   {{- end }}
-  - apiGroups: [ "" ]
-    resources: [ "persistentvolumes" ]
-    verbs: [ "get", "list", "watch", "create", "delete", "patch" ]
-  - apiGroups: [ "" ]
-    resources: [ "persistentvolumeclaims" ]
-    verbs: [ "get", "list", "watch", "update" ]
-  - apiGroups: [ "storage.k8s.io" ]
-    resources: [ "storageclasses" ]
-    verbs: [ "get", "list", "watch" ]
-  - apiGroups: [ "storage.k8s.io" ]
-    resources: [ "volumeattachments" ]
+  - apiGroups: [""]
+    resources: ["persistentvolumes"]
+    verbs: ["get", "list", "watch", "create", "delete", "update", "patch"]
```

Comment:
Please ensure that the verbs are consistent across the drivers. I see us making changes elsewhere: https://github.com/dell/helm-charts/pull/685/files. Thanks!

Reply:
Sure @AronAtDell, we are making sure the changes are consistent across drivers. In this PR - https://github.com/dell/helm-charts/pull/685/files, that specific change is required only for Replication. That change is reverted now since that PR is for driver specific changes. Replication specific changes will be done as part of different PR.